bindings/matrix-sdk-crypto-ffi/src/olm.udl

```diff
@@ -282,7 +289,7 @@ interface OlmMachine {
     [Throws=CryptoStoreError]
     Device? get_device([ByRef] string user_id, [ByRef] string device_id, u32 timeout);
     [Throws=CryptoStoreError]
-    void mark_device_as_trusted([ByRef] string user_id, [ByRef] string device_id);
```

Comment:
We'll need to notify people on the Android team since this is a breaking change. @BillCarsonFr do I notify you or someone else?